Vesper-Agents-Suite/Compound-Collateral-Factor-Agent/src/index.spec.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  Finding, FindingSeverity, FindingType, HandleTransaction, TransactionEvent
+} from "forta-agent";
+import { TestTransactionEvent } from "forta-agent-tools";
+import Web3 from 'web3';
+import agent from ".";
+import { COLLATERAL_FACTOR_EVENT_ALERT_ID, COMPOUND_COMPTROLLER_ADDRESS, NEW_COLLATERAL_FACTOR_SIGNATURE } from "./utils";
+
+describe("compount detect update to collateral factor mantissa", () => {
+  let handleTransaction: HandleTransaction
+  let web3: Web3
+  const mockAddress = '0xB44ddb00B19a2F20E3c1C31A14C8965D75a8e4De'
+
+  beforeAll(() => {
+    handleTransaction = agent.handleTransaction
+    web3 = new Web3()
+  })
+
+  describe("handleTransaction", () => {
+    it("returns a finding if collateral update event has occured", async () => {
+      const data = web3.eth.abi.encodeParameters(
+        ['address', 'uint256', 'uint256'],
```

Comment:
use abi methods exported from `forta-agent-tools` to avoid setting up web3 for encrytpion/decryption purposes.